factory/pom.xml

```diff
@@ -165,6 +159,12 @@
               <artifactId>auto-value</artifactId>
               <version>${auto-value.version}</version>
             </path>
+            <path>
+              <groupId>net.ltgt.gradle.incap</groupId>
+              <artifactId>incap</artifactId>
+              <version>0.3</version>
+              <scope>provided</scope>
```

Comment:
That should be removed (Maven is lenient and won't complain, but it's useless and looks weird here)